indico/web/templates/forms/time_widget.html

```diff
@@ -4,7 +4,7 @@
     <div class="time-widget">
         {# the hidden field has autofocus on purpose to prevent our JS from focusing the time picker #}
         <input type="hidden" name="{{ field.name }}" id="{{ field.id }}-timestorage" autofocus
-               {% if field.data is not none %}value="{{ field.data.strftime('%H:%M') }}"{% endif %}
+               {% if field.data is not none %}value="{{ field.data | format_time }}"{% endif %}
```

Comment:
Since this is a value that's consumed programmatically I think we should be explicit about the format, so I'd stick with strftime or use the `'code'` format of `format_time`)

Reply:
Yep, make sense :+1: